custom/config-parser.md

```diff
@@ -41,7 +41,7 @@ export default definePreparserSetup(({ filepath, headmatter }) => {
 
 This example systematically replaces any `@@@` line by a line with `hello`. It illustrates the structure of a preparser configuration file and some of the main concepts the preparser involves:
 - `definePreparserSetup` must be called with a function as parameter.
-- The function receives the file path (of the root presentation file) and headmatter (from the md file). It could use this information (e.g., enable extensions based on the presentation file).
+- The function receives the file path (of the root presentation file), the headmatter (from the md file) and a mode (dev, build or export). It could use this information (e.g., enable extensions based on the presentation file or whether we are exporting a PDF).
```

Comment:
We could mention the version here instead if you think it's a concern. Putting up front might feel a bit confusing as ppl don't have the context yet.

Reply:
Good idea.